src/components/CaseStudies/AuthorBio.tsx

```diff
@@ -1,23 +1,62 @@
-export const AuthorBio: React.FunctionComponent<{
-    customer: string
+interface Author {
     image?: string
-    author: string
+    name: string
     title: string
+}
+
+export const AuthorBio: React.FunctionComponent<{
+    customer: string
     about: string
-}> = ({ customer, image, author, title, about }) => (
-    <div className="d-flex flex-column flex-md-row align-items-start">
-        {image && (
-            <div className="col-md-3">
-                <img className="rounded-circle max-w-150" src={image} alt={author} />
+    authors: Author[]
+}> = ({ customer, about, authors }) => (
+    <section>
+        {authors.length > 1 ? (
+            // Multi author design
+            <div>
+                <section className="col-lg-6 mx-auto p-0 max-w-650">
+                    <h5 className="mb-4 text-xl font-weight-bold">About {customer}</h5>
+                    <p>{about}</p>
+                </section>
+
+                <section className="d-md-flex d-block justify-content-between col-lg-9 mx-auto mt-6 p-0">
+                    {authors.map(a => (
+                        <div key={a.name}>
```

Comment:
Looks like the author sections have some bottom spacing between them. We should add a bottom margin here on mobile when they stack, except for the last one so it doesn't push the section further than it needs to.

Just to note, in Tailwind after #5378 gets merged, you can do this mobile-first by first setting the mobile styles, and then override the breakpoint when they align horizontally on tablet/desktop. (ie: `tw-mb-sm md:tw-mb-0`) The `xs/sm/md/lg/xl` spacings are defined from our DLS, but you could also use numbers (ie: `tw-mb-04`). The breakpoint modifiers come first with a colon that references the screen size (ie: `md:tw-mb-0` would reference the `md` breakpoint and set the bottom margin to `0`).

You can add these when `index !== index.length - 1`